scripts/test_iroha2_java_api.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+mkdir iroha_api_test
+cd iroha_api_test || exit
+git clone https://github.com/hyperledger/iroha.git
+cd iroha || exit
+git checkout iroha2-java
+./gradlew build
```

Comment:
Can we use already built and published artifacts for iroha2-java?

Reply:
Normally, Jar file doesn't include dependencies and tests, so we need a Java project to assemble all dependencies and to run the tests. Then it is better to create a different jar with tests classes to run them. It looks like a separate Java project to test Iroha2 API. Since the iroha2-java project is small, it takes a little time to build it (17s in total with test run). I think the current approach is good enough.

Reply:
Usage of built iroha2 docker image can save time.